fly.beta.toml

```diff
@@ -0,0 +1,39 @@
+app = 'lebkuchen-fm-beta'
+primary_region = 'ewr'
```

Comment:
ewr? 🤔

Reply:
Since the beginning of this year we're running in ewr region which is on the east coast of the USA. This is because Google Gemini is not available in the EU.